client/src/Components/Questions.js

```diff
@@ -92,124 +105,122 @@ const Questions = (props) => {
-							onChange={checkHandler}
-							id="answer_a"
-							checked={isChecked && radioId == "answer_a" ? true : false}
-						/>
-					) : null}
-					{answer_a ? <Label for="answer_a">{answer_a}</Label> : null}
-				</FormGroup>
+		<div>
+			<Form  className="survey-form"  onSubmit={submitHandler}>
 
-				<FormGroup className="answers">
-					{answer_b ? (
-						<Input
-							type="radio"
-							name="answer"
-							value={answer_b}
-							onChange={checkHandler}
-							id="answer_b"
-							checked={isChecked && radioId == "answer_b" ? true : false}
-						/>
-					) : null}
-					{answer_b ? <Label for="answer_b">{answer_b}</Label> : null}
-				</FormGroup>
-				<FormGroup className="answers">
-					{answer_c ? (
-						<Input
-							type="radio"
-							name="answer"
-							value={answer_c}
-							onChange={checkHandler}
-							id="answer_c"
-							checked={isChecked && radioId == "answer_c" ? true : false}
+				<FormGroup >
+					<FormGroup className="answers">
+						<input
+							type="text"
+							placeholder="Enter your name"
+							onChange={changeHandler}
+							className="answers"
+							required
+							autoFocus
 						/>
-					) : null}
-					{answer_c ? <Label for="answer_c">{answer_c}</Label> : null}
-				</FormGroup>
-				<FormGroup className="answers">
-					{answer_d ? (
-						<Input
-							type="radio"
-							name="answer"
-							value={answer_d}
-							onChange={checkHandler}
-							id="answer_d"
-							checked={isChecked && radioId == "answer_d" ? true : false}
-						/>
-					) : null}
-					{answer_d ? <Label for="answer_d">{answer_d}</Label> : null}
-				</FormGroup>
-				<FormGroup className="answers">
-					{answer_e ? (
-						<Input
-							type="radio"
-							name="answer"
-							value={answer_e}
-							onChange={checkHandler}
-							id="answer_e"
-							checked={isChecked && radioId == "answer_e" ? true : false}
-						/>
-					) : null}
-					{answer_e ? <Label for="answer_e">{answer_e}</Label> : null}
-				</FormGroup>
-				<FormGroup className="answers">
-					{answer_f ? (
-						<Input
-							type="radio"
-							name="answer"
-							value={answer_f}
-							onClick={checkHandler}
-							id="answer_f"
-							checked={isChecked && radioId == "answer_f" ? true : false}
-						/>
-					) : null}
-					{answer_f ? <Label for="answer_f">{answer_f}</Label> : null}
+						{questionData.question_code ? (
+							<ReactMarkdown className="code">
+								{questionData.question_code}
+							</ReactMarkdown>
+						) : null}
+						<ReactMarkdown>{questionData.question}</ReactMarkdown>
+					</FormGroup>
+					<FormGroup className="answers">
+						{answer_a ? (
+							<Input
+								type="radio"
+								name="answer"
+								value={answer_a}
+								onChange={checkHandler}
+								id="answer_a"
+								checked={isChecked && radioId == "answer_a" ? true : false}
+							/>
+						) : null}
+						{answer_a ? <Label for="answer_a">{answer_a}</Label> : null}
```

Comment:
This could go within the same block above (but you'll need to surround them with `<>` and `</>` so JSX doesn't complain about multiple sibling elements.

Reply:
done. i put them inside formgroup to work